### PR TITLE
Build/search,create label feature - update/auto delete feature - Replace NPM with Yarn

### PR DESCRIPTION
**Pull request by mason-nguyen-goldenowl**
**Task/Ticket:**
Build search page, feature - Update auto delete feature - Create label feature - Replace NPM with Yarn
**Description:**

- Build UI for Search page

- Build search feature

- Build UI for Edit label component

- Build create label feature

- Update auto delete after 7 days feature

...
Screenshots
Screenshot from 2022-01-10 17-19-00
Screenshot from 2022-01-10 17-19-11
Screenshot from 2022-01-10 17-19-44
:

### CODE REVIEW

**Comment by mason-nguyen-goldenowl** [comment]
@jun-nguyen-goldenowl conflicts have been fixed

File: src/pages/archive/Archive.jsx [hunk 1]
```diff
@@ -13,8 +12,9 @@ export default function Archive() {
 
   const { arrArchive } = useSelector((state) => state.note);
   const renderNoteCard = () => {
-    return arrArchive?.map((note, index) => {
-      return <NoteCard content={note} key={index} />;
+    return arrArchive?.map((note) => {
+      return <NoteCard content={note} key={note.id} />;
+
```
Comment (jun-nguyen-goldenowl) [inline src/pages/archive/Archive.jsx:17]: remove this redundant line. 
I think you have formatting configuration which leads to additional blank line like this

File: src/pages/trash/Trash.jsx [hunk 2]
```diff
@@ -37,7 +37,9 @@ export default function Trash() {
         </div>
         <div className="right">
           <div className="trash__dr">
-            <p>Notes in Trash are deledted after 7 days</p>
+
+            <p>Notes in Trash are deleted after 7 days</p>
+
```
Comment (jun-nguyen-goldenowl) [inline src/pages/trash/Trash.jsx:42]: remove redundant blank line

File: src/redux/type/NoteType.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (jun-nguyen-goldenowl) [inline src/redux/type/NoteType.js:1]: no need to add blank line at the 1st line of the file. Usually, a blank line at the bottom is accepted

**Review by jun-nguyen-goldenowl — APPROVED** [review]

**Pull request merged**